tflint-ruleset-blueprint/rules/terraform_required_version.go

```diff
@@ -48,7 +54,30 @@ const (
 
 // Checks if a module has a terraform required_version within valid range.
 func (r *TerraformRequiredVersion) Check(runner tflint.Runner) error {
-	splitVersion := strings.Split(minimumTerraformRequiredVersion, ".")
+	config := &TerraformRequiredVersionConfig{}
+	if err := runner.DecodeRuleConfig(r.Name(), config); err != nil {
+		return err
+	}
+
+	minVersion := minimumTerraformRequiredVersion
+	if config.MinVersion != "" {
+		if _, err := version.NewSemver(config.MinVersion); err != nil {
+			return err
+		} else {
```

Comment:
nit: since there is return above, I think we can omit the else?

```
if _, err := .. {
return err
}
minVersion = config.MinVersion
```

Reply:
Yup, originally I had the order reversed :)